schema-supplement.rst

```diff
@@ -52,40 +57,18 @@ instead of, or in addition to, the canonical package name.
 :attribute:`license`
 --------------------
 
-:Type: Special
+:Type: :type:`string`
 :Applies To: :object:`package`, :object:`component`
 
 Specifies the license or licenses
 under which the package is distributed.
-A :type:`string` value shall be used for a single license.
-If multiple licenses need to be specified,
-a :type:`list` shall be used.
-Each :type:`string` value of the :type:`list`
-is a license which *always* applies to the package.
-If a package is wholly or partly multi-licensed
-(that is, the user has a choice of license),
-the top level :type:`list` shall contain a :type:`list` of values
-representing possible licenses.
-For example, the value ``["CC-BY-4.0", ["GPL-2.0", "LGPL-3.0+"]]``
-indicates that some portions of the package
-are licensed under :string:`"CC-BY-4.0"`,
-while others are licensed as (at the user's choice)
-either :string:`"GPL-2.0"` or :string:`"LGPL-3.0+"`.
-If necessary, further nesting may be employed;
-each nesting level alternates
-between inclusive ("A **and** B")
-and exclusive ("A **or** B") licensing.
-
+A :type:`string` value shall be a well formed
+|SPDX|_ `License List`_ expression.
```

Comment:
```suggestion
|SPDX|_ `License Expression`_.
```

...which should probably link to https://spdx.github.io/spdx-spec/v2.3/SPDX-license-expressions/.

Similar for `default_license`.

On that note, I wonder if we need a `license_meta_version`/`license_spdx_version` attribute as well? (Otherwise, I suppose we can declare that the SPDX version is a function of the CPS version.)

Reply:
I would agree that we should do one of those things. I see advantages and disadvantages to each

Reply:
If there's a preference one way or another I can update this PR with that.